assets/company-admin/components/CompanyAdminApp.tsx

```diff
@@ -184,26 +217,32 @@ class CompanyAdminAppComponent extends React.Component<any, any> {
                                     </div>
                                 </section>
                             </div>
-                            {this.props.userToEdit == null ? null : (
-                                <EditUser
-                                    original={this.props.usersById[this.props.userToEdit._id] || {}}
-                                    user={this.props.userToEdit}
-                                    onChange={this.props.editUser}
-                                    errors={this.props.errors}
-                                    companies={this.props.companies}
-                                    onSave={this.saveUser}
-                                    onResetPassword={this.props.resetPassword}
-                                    onClose={this.props.closeUserEditor}
-                                    onDelete={this.deleteUser}
-                                    resendUserInvite={this.props.resendUserInvite}
-                                    currentUser={this.props.user}
-                                    products={this.props.products}
-                                    hideFields={getConfig('allow_companies_to_manage_products') ?
-                                        ['company'] :
-                                        ['company', 'sections', 'products']
-                                    }
-                                />
-                            )}
+                            {
+                                this.props.userToEdit !== null && (
+                                    <EditUser
+                                        original={this.props.usersById[this.props.userToEdit._id ?? ''] || {}}
```

Comment:
Don't default to an empty string. Maybe the idea is that when you're creating a new user, it is `null`. That would make sense to keep it then. If you convert it to a string there would no longer be a way to differentiate between a real ID and ID missing. Unless you check the length, but that's not a good approach.